apps/prairielearn/src/ee/institution/admin/course.ts

```diff
@@ -0,0 +1,37 @@
+import { loadSqlEquiv, queryRow, queryRows } from '@prairielearn/postgres';
+import { Router } from 'express';
+import asyncHandler = require('express-async-handler');
+import { CourseInstanceSchema, CourseSchema } from '../../../lib/db-types';
+import { InstitutionAdminCourse } from './course.html';
+import { getInstitution } from '../utils';
+
+const sql = loadSqlEquiv(__filename);
+const router = Router({ mergeParams: true });
+
+router.get(
+  '/',
+  asyncHandler(async (req, res) => {
+    const institution = await getInstitution(req.params.institution_id);
```

Comment:
I've found that the `selectAndAuthz...` functions in PT do a nice job of combining authz with data fetching. There is certainly no need to shift to that model here, however, unless you want to. For this page both the `course` and `institution` would be fetched by `selectAndAuthz...` functions and only `courseInstances` would need a dedicated page query.

Reply:
I'll do that once we implement institution admins. For now, this entire set of pages is gated on the user being a global admin, so auth within pages isn't strictly necessary (though I've been including `institution_id` in queries for future-proofing and safety to make sure that entities do in fact belong to the institution being viewed).

Reply:
Yep, makes sense.